drivetrain.py

```diff
@@ -43,7 +43,10 @@ def set_neutral(self):
         self.set_servo_pulse(self.channels['right'], self.servo_mid)
 
     # TODO - flesh out setters for raw pulse values (both channels)
-    def mix_channels(self, pulse_throttle, pulse_steering):
+    def mix_channels(self, throttle, steering):
```

Comment:
Needs documentation stating that this method mixes throttle and steering but that it also passes those values directly to the motors.